.env.example

```diff
@@ -6,3 +6,6 @@ DAPP_DID_MNEMONIC=
 DAPP_DID_URI=
 DAPP_NAME=
 JWT_SIGNER_SECRET=
+CTYPE_HASH=
```

Comment:
I see this file having already some defaults. If we want to go with default values (which I don't agree with), why not putting them here directly?